packages/mask/src/plugins/Tips/storage/index.ts

```diff
@@ -1,13 +1,16 @@
 import { remove } from 'lodash-unified'
-import type { ScopedStorage } from '@masknet/shared-base'
+import type { ScopedStorage, EnhanceableSite } from '@masknet/shared-base'
 import { NonFungibleToken, isSameAddress } from '@masknet/web3-shared-base'
 import type { ChainId, SchemaType } from '@masknet/web3-shared-evm'
+import { useSubscription } from 'use-subscription'
 
 interface StorageValue {
     addedTokens: Array<NonFungibleToken<ChainId, SchemaType>>
+    userGuide: Record<EnhanceableSite, number>
 }
 
 export const storageDefaultValue = {
+    userGuide: {} as Record<EnhanceableSite, number>,
     publicKey: null as null | string,
     addedTokens: [],
 }
```

Comment:
```suggestion
export const storageDefaultValue: StorageValue = {
    userGuide: {},
    publicKey: null as null | string,
    addedTokens: [],
}
```

Reply:
When setup default value, the `setupStore` method will check the `storageDefaultValue ` type. The  `userGuide: {},` will get type error.

Reply:
What if we change `StorageValue` to:

```
interface StorageValue {
    addedTokens: Array<NonFungibleToken<ChainId, SchemaType>>
    userGuide: Partial<Record<EnhanceableSite, number>>
}
```